scripts/create-action-plan-selectors/create-action-plan-selectors.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import sys
+
+import requests
+from sqlalchemy import create_engine
+from structlog import wrap_logger
+
+from config import Config
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_casetypeoverrides():
+    logger.debug('Retrieving casetypeoverrides')
+    engine = create_engine(Config.DATABASE_URI)
+    connection = engine.connect()
+
+    sql = "SELECT o.sampleunittypefk, o.actionplanid, c.exerciseref, c.survey_uuid " \
```

Comment:
@andrewmil probably worth adding a comment explaining what the query is doing, also maybe docstrings for the methods in the file

Reply:
Good point I'll add the comment
Will probably miss docstrings though since this is probably just a one use script